### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dombioinfo**
Add a new tile to manage configuration
Change notification system

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,6 +13,43 @@
         type="text/javascript"></script>
     <script src="http://code.jquery.com/mobile/git/jquery.mobile-git.js"
         type="text/javascript"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.min.js" type="text/javascript"></script>
```
Comment (pypingou) [inline index.html]: We shouldn't pull from CDNs, let's just put the JS file in the sources w/ us

Comment (dombioinfo) [inline index.html]: Ok. but and we do the same for jquery.mobile, we don't ?

**Comment by pypingou** [comment]
Let me know when you're ok to merge this :)

File: index.html (hunk 1 above)
Comment (pypingou) [inline index.html]: You're right, ideally we shouldn't do it either, especially considering I've been bitten by changes made to it a couple of times

**Comment by dombioinfo** [comment]
It's OK

**Comment by pypingou** [comment]
Let's do it then :)

**Pull request merged**